src/marbl_interior_tendency_mod.F90

```diff
@@ -1415,6 +1415,7 @@ subroutine compute_autotroph_calcification(km, autotroph_local, temperature, car
     !-----------------------------------------------------------------------
     !  local variables
     !-----------------------------------------------------------------------
+    real(r8), parameter :: picpoc_temp_threshold = 10000._r8**(1._r8/3._r8)
```

Comment:
We should change this to include 0s in the tenths place of each number:

```
real(r8), parameter :: picpoc_temp_threshold = 10000.0_r8**(1.0_r8/3.0_r8)
```

We adopted that convention after you had already started this branch, sorry!